metricbeat/module/traefik/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: traefik
+  title: "traefik"
+  description: >
+    experimental[]
```

Comment:
I think we now generate this automatically if the release setting is not set. So you can skip it here.